example-app/SantokuApp/src/apps/app/use-cases/useAppInitialize.ts

```diff
@@ -62,7 +62,7 @@ export const useAppInitialize = () => {
   const initialize = useCallback(async () => {
     await initializeCoreFeatures();
     // CsrfTokenを取得し、AxiosInstanceのデフォルトヘッダに設定
-    await refreshCsrfToken();
+    // await refreshCsrfToken();
```

Comment:
What is the reason for commenting out?

Reply:
Sorry mistake.
265415f9e026b259843a96fba1ca03617b0a3797